mb-url-http.el

```diff
@@ -149,6 +149,24 @@ EVT describes the type of event."
           (t
            (format "%s:%s" name value)))))
 
+(defun mb-url-http-headers-list ()
```

Comment:
Append more headers with reference to: https://github.com/emacs-mirror/emacs/blob/emacs-28/lisp/url/url-http.el#L327